doc/attributes.md

```diff
@@ -67,3 +67,47 @@ message:
 
 When an attribute is used in Istio, its name is given along with a value. The value must be of the type declared by the corresponding descriptor. This
 type checking makes it possible for the Istio system to statically detect or prevent many configuration errors.
+
+# Standard Istio Attribute Vocabulary
+
+In a standard Istio deployment (using the Istio proxy), the system will produce the following set of attributes. This is considered the canonical attribute set for Istio.
+
+| Name | Type | Description | Kubernetes Example |
+|------|------|-------------|--------------------|
+| source.ip | ip_address | The IP address of the client that sent the request. | 10.0.0.117 |
+| source.port | int64 | The port on the client that sent the request. | 9284 |
```

Comment:
Actually, that feels a bit heavy weight. Filing a single issue about attribute vocab and I'll call out individual attributes there.